mlflow/server/handlers.py

```diff
@@ -4046,6 +4077,7 @@ def _get_dataset_records_handler(dataset_id):
     SetTraceTagV3: _set_trace_tag_v3,
     DeleteTraceTagV3: _delete_trace_tag,
     LinkTracesToRun: _link_traces_to_run,
+    BatchGetTraces: _batch_get_traces,
```

Comment:
Why don't we use the existing naming convention like `(Get|List) + {Entity name} + s`?

Reply:
It's following batch_get_traces API naming